examples/README.md

```diff
@@ -2,7 +2,7 @@
 | Domain | Example | Dataset | Framework |
 |:------:|:-----:|:-------:|:------------------:|
 | Tutorials | <p>[mnist_pytorch](tutorials/mnist_pytorch)<br>[fashion_mnist_tf_keras](tutorials/fashion_mnist_tf_keras)</p> | <p>MNIST<br>Fashion MNIST</p> | <p>PyTorch<br>TensorFlow (tf.keras)</p> |
-| Computer Vision | <p>[cifar10_pytorch](computer_vision/cifar10_pytorch)<br>[mnist_multi_output_pytorch](computer_vision/mnist_multi_output_pytorch)<br>[fasterrcnn_coco_pytorch](computer_vision/fasterrcnn_coco_pytorch)<br>[mnist_estimator](computer_vision/mnist_estimator)<br>[cifar10_tf_keras](computer_vision/cifar10_tf_keras)<br>[iris_tf_keras](computer_vision/iris_tf_keras)<br>[unets_tf_keras](computer_vision/unets_tf_keras)</p> | <p>CIFAR-10<br>MNIST<br>Penn-Fudan Dataset<br>MNIST<br>CIFAR-10<br>Iris Dataset<br>Oxford-IIIT Pet Dataset</p> | <p>PyTorch<br>PyTorch<br>PyTorch<br>TensorFlow (Estimator API)<br>TensorFlow (tf.keras)<br>TensorFlow (tf.keras)<br>TensorFlow (tf.keras)</p> |
+| Computer Vision | <p>[cifar10_pytorch](computer_vision/cifar10_pytorch)<br>[mnist_multi_output_pytorch](computer_vision/mnist_multi_output_pytorch)<br>[fasterrcnn_coco_pytorch](computer_vision/fasterrcnn_coco_pytorch)<br>[mnist_estimator](computer_vision/mnist_estimator)<br>[iris_tf_keras](computer_vision/iris_tf_keras)<br>[unets_tf_keras](computer_vision/unets_tf_keras)<br>[mnist_tf_layers](computer_vision/mnist_tf_layers)</p> | <p>CIFAR-10<br>MNIST<br>Penn-Fudan Dataset<br>MNIST<br>CIFAR-10<br>Iris Dataset<br>Oxford-IIIT Pet Dataset<br>MNIST</p> | <p>PyTorch<br>PyTorch<br>PyTorch<br>TensorFlow (Estimator API)<br>TensorFlow (tf.keras)<br>TensorFlow (tf.keras)<br>TensorFlow (tf.keras)<br>TensorFlow (tf.layers)</p> |
```

Comment:
Should the framework here be TensorFlow (Estimator API) instead of tf.layers since we do not support that. If so, can we move it next to the other Estimator API example `mnist_estimator`?

Reply:
What about `TensorFlow (tf.layers via Estimator API)` or something?  Calling this an estimator example is really missing the point of the example.

Reply:
Yeah I like that -- can we move it next to the other Estimator example placement wise in the Computer Vision category?

Reply:
sure